agents/sorenson_dlm/dlm_agent.py

```diff
@@ -74,7 +77,7 @@ def read_current(self):
         msg = self.rec_msg()
         return msg
 
-    def sys_err_check(self):
+    def sys_err_check(self):  # never used
```

Comment:
I will keep this in, in case it is needed in the future. It could be useful for debugging purposes